oak_functions/sdk/test_utils/src/lib.rs

```diff
@@ -257,12 +257,11 @@ pub fn assert_response_body(response: Response, expected: &str) {
     )
 }
 
-// Create some valid wasm bytecode
-pub fn create_some_wasm_module_bytes() -> Vec<u8> {
+/// Create some valid Wasm bytecode only calling Abi functions `read_request` and `write_request`.
+pub fn create_echo_wasm_module_bytes() -> Vec<u8> {
```

Comment:
Is not it better to have this function in the test module itself? Or do you think that this function will be used in many tests? 

I feel like this module should contain generic test functionality, and this function suddenly feels very specific.

Reply:
Hmmm... I wanted it to be visible for other tests.

If you just need some valid Wasm bytecode (e.g., for creating a WasmHandler, but not executing the code), I think it could be useful.

I was looking for this function in the test utils before, so I put it here. But happy to move it, if you are not convinced.

Reply:
We have similar code to this repeated in several places in `oak_functions`. It might be better to change this to a more generic function like `create_wasm_module_bytes(manifest_path: &str, release: bool)` and use that everywhere where we use `compile_rust_wasm`. For instance: https://github.com/project-oak/oak/blob/201977e66a923a6a74cb5217bff6422423042bc2/oak_functions/loader/src/tests.rs#L114-L119

WDYT?

(This can be done in a separate PR.)

Reply:
Ah, but I think the main benefit of `create_echo_wasm_bytes` is that we _fix_ the manifest path to the `Echo` module. 

So if---in future tests---we want to just get some valid Wasm code, we can use that. But mayb it is to confusing. 
Do you think it would be better to remove it from utils?

Do you think we gain much from `create_wasm_module_bytes`? I am not too sure, but happy to add an issue :)

Reply:
I don't see a big difference between `create_echo_wasm_bytes()` and `create_wasm_module_bytes("examples/echo/module/Cargo.toml", false)`, but if we have `create_wasm_module_bytes` we can use it in other places too. 

Anyways, we can perhaps add `create_wasm_module_bytes` in a separate PR, and decide if we want to replace `create_echo_wasm_bytes` too.